src/omegga/player.ts

```diff
@@ -344,10 +344,10 @@ class Player implements OmeggaPlayer {
     );
   }
 
-  async getPawn(): Promise<string> {
+  async getPawn(): Promise<string | undefined> {
```

Comment:
This should probably return `string | null` instead (so the return becomes `return null;`)